lib/node_modules/@stdlib/array/bool/benchmark/benchmark.map.js

```diff
@@ -0,0 +1,55 @@
+/**
```

Comment:
This file can now be updated by merging in the latest `develop` and using `isBooleanArray`.